### PR TITLE
tree intersection

### PR DESCRIPTION
**Pull request by sarastrasner**
- [X] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [X] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [X] Link to code
     - [X] Picture of whiteboard

### CODE REVIEW

File: javascript/treeIntersection/tree-intersection.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+const HashMap = require('../hashtable/hashtable');
+
+function treeIntersection(treeOne, treeTwo) {
+  let outputArray = [];
+  if (!treeOne.root || !treeTwo.root) {
+    throw new Error();
```
Comment (adrianhuebner) [inline javascript/treeIntersection/tree-intersection.js:7]: This line is causing an error based on logic from your repeated word hash table that you are calling in your test
